cypress/integration/userCanSeeFirstHeader.feature.js

```diff
@@ -0,0 +1,9 @@
+describe("user views menus", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
+  });
+
+  it("displays first header header", () => {
+    cy.get("h1").should("contain", "VAMOS TEAM 2");
```

Comment:
Can you add picture of the test passing?

Reply:
Done. Check PR.